.travis.yml

```diff
@@ -57,6 +57,14 @@ script:
   - if [ $REPO_PROJ == "qemu_v8" ]; then cd $HOME/$REPO_PROJ/qemu && git submodule update --init dtc; fi
   # Here we are using Travis environment variables to select the correct branch
   # based on either a pull request or a normal push.
-  - cd $HOME/$REPO_PROJ/build && git remote add github https://github.com/$TRAVIS_REPO_SLUG && git remote update
-  - cd $HOME/$REPO_PROJ/build && git checkout github/$TRAVIS_BRANCH
+  - |
+        if [ "$TRAVIS_EVENT_TYPE" == "pull_request" ]; then
```

Comment:
What type of event will we receive, if source branch will be updated in already existing PR (it's not clear from from description here https://docs.travis-ci.com/user/environment-variables/)? I guess it will be also `"pull_request"`, right?